src/App.tsx

```diff
@@ -17,18 +45,25 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setQuery={setQuery}
```

Comment:
I remember Denys told that we should create methods for 'set' and don't provide this directly
But I forgot why -_-